std/os/index.zig

```diff
@@ -758,21 +760,25 @@ pub fn getEnvPosix(key: []const u8) ?[]const u8 {
 pub const GetEnvVarOwnedError = error{
     OutOfMemory,
     EnvironmentVariableNotFound,
+    InvalidUtf8,
```

Comment:
The changes to `getEnvVarOwned` look good. Can you put a link to #1774 in the doc comments for this `InvalidUtf8` error? I think once that issue is solved, this function will not have any utf8 encoding errors possible.

Reply:
Added